src/config.js

```diff
@@ -1 +1,14 @@
-export { default } from "./configs/config.aws.js";
+// The build script injects this global variable if "local.config.js"
+// exists in the project root (alongside package.json). That file is in
+// gitignore and so can be modified without making changes to this file.
```

Comment:
Well, should be added to gitignore :)

Reply:
.gitignore added in cf49208

Ultimately I'd like to come up with a scheme where the demo map has a user-selectable list of tileservers that a user can swap out on the fly.  But I think this does the job for now.